src/compat/db_query.c

```diff
@@ -3150,6 +3150,9 @@ db_query_tuple_count (DB_QUERY_RESULT * result)
 int
 db_query_column_count (DB_QUERY_RESULT * result)
 {
+  DB_QUERY_TYPE *t;
+  int num_attrs = 0;
```

Comment:
variable name would be better "num_cols" than "num_attrs"

Reply:
Thank you, I accept.